clients/docs/rust.rst

```diff
@@ -0,0 +1,122 @@
+.. _client-examples-rust:
+
+Rust
+====
+
+In this tutorial, you will run a Rust client application that produces messages
+to and consumes messages from an |ak-tm| cluster.
+
+.. include:: includes/client-example-overview.rst
+
+
+Prerequisites
+-------------
+
+Client
+~~~~~~
+
+-  `Rust client for Apache
+   Kafka <https://github.com/fede1024/rust-rdkafka#installation>`__ installed on
+   your machine
+
+Kafka Cluster
+~~~~~~~~~~~~~
+
+.. include:: includes/client-example-prerequisites.rst
+
+
+Setup
+-----
+
+#. .. include:: includes/clients-checkout.rst
+
+#. Change directory to the example for Rust.
+
+   .. code-block:: bash
+
+      cd clients/cloud/rust/
+
+#. .. include:: includes/client-example-create-file-librdkafka.rst
+
+
+Basic Consumer and Producer
+---------------------------
+
+.. include:: includes/producer-consumer-description.rst
+
+
+Produce Records
+~~~~~~~~~~~~~~~
+
+#. Build the producer and consumer binaries:
+
+   .. code-block:: bash
+
+      cargo build
```

Comment:
Let's split this out:

* user runs `cargo build`
* expected output is the next two lines (compiling...finished)

Reply:
Updated